src/api/admin.js

```diff
@@ -0,0 +1,69 @@
+import BASEURL from "./index";
+
+export const fetchCreateConsoleListing = async ({
+  title,
+  description,
+  year,
+  price,
+}) => {
+  try {
+    const response = await fetch(`${BASEURL}/consoles`, {
+      method: "POST",
+      headers: {
+        "Content-type": "Application/json",
```

Comment:
Im not sure this is an issue, but case sensitivity on "Application/json" could possibly present a problem vs. application/json